README.md

```diff
@@ -1 +1,5 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
```

Comment:
- Provide a README fully described you can use the [template](https://github.com/microverseinc/readme-template) provided by Microverse.